src/Microsoft.AspNet.Hosting.Abstractions/EnvironmentName.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Hosting
+{
+    public static class EnvironmentName
```

Comment:
IMHO when looking at the code consuming the members, it looks like the declaration could also be an `enum`. So the guideline regarding (non-)`[Flags]` could be applied which makes singular the right choice.